### DISCUSSION TITLE
dimension don't match  error after resampling

### DISCUSSION THREAD

**Discussion by coralraha1234** [gmtsar/gmtsar#784] [Q&A] [answered]
hi the process step-1 and 2 and 3 is done. In step 4 i  faced with this error  "dimension don't match! ../2021108_2021132/corr.grd   then i used resample.csh but after using resample.csh i faced with this error  dimension don't match! ../2021108_2021132/corrcutt.grd 

`ompute RMS misfit
compute DEM error

setting smoothing to   1.000 
setting radar wavelength to   0.055 m 
setting radar incidence angle to  39.351 degree 
setting range to 845450.236 m 
Memory Allocation Successful...
read table file ...
number of SAR scenes is 32 
number of interferograms is 124 
read phase and correlation grids ...
 dimension don't match! ../2021108_2021132/corrcutt.grd 
ln: failed to create symbolic link './trans.dat': File exists
proj_ra2ll.csh
grd2xyz [ERROR]: File vel.grd not found
`

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
Cold you run a `gmt grdinfo` on your `../2021108_2021132/corrcutt.grd` and your `../2021108_2021132/unwrap.grd` and compare their dimensions with you input dimensions for the sbas program?

**Comment by coralraha1234** [comment]
On Monday, August 14, 2023 at 04:13:48 AM GMT+3:30, Xiaohua(Eric) Xu ***@***.***> wrote:  
 
 


Cold you run a gmt grdinfo on your ../2021108_2021132/corrcutt.grd and your ../2021108_2021132/unwrap.grd and compare their dimensions with you input dimensions for the sbas program?

—
Reply to this email directly, view it on GitHub, or unsubscribe.

You are receiving this because you authored the thread.

Hi first of all I appreciate your help of course i will do it and I will send you the result thank you so much Message ID: ***@***.***>

**Comment by coralraha1234** [comment]
On Monday, August 14, 2023 at 04:13:48 AM GMT+3:30, Xiaohua(Eric) Xu ***@***.***> wrote:  
 
 


Cold you run a gmt grdinfo on your ../2021108_2021132/corrcutt.grd and your ../2021108_2021132/unwrap.grd and compare their dimensions with you input dimensions for the sbas program?

—
Reply to this email directly, view it on GitHub, or unsubscribe.

You are receiving this because you authored the thread.
***@***.***:/repository/33269/urmia2021-2023/merge/2021108_2021132$ gmt grdinfo -C corr.grd | awk '{print ***@***.***:/repository/33269/urmia2021-2023/merge/2021108_2021132$ gmt grdinfo -C corr.grd | awk '{print $11}'6562
.....and after resample.csh 

***@***.***:/repository/33269/urmia2021-2023/merge/2021108_2021132$ gmt grdinfo -C corrcutt.grd | awk '{print ***@***.***:/repository/33269/urmia2021-2023/merge/2021108_2021132$ gmt grdinfo -C corrcutt.grd | awk '{print $11}'8565

sbas intfc.tab scene.tab 124 32 8565 6561  -smooth 1.0 -wavelength 0.0554658 -incidence 39.35061651713706 -range 845450.236362 -rms -dem


oh my god maybe i must change resample size  in  vice versa  because i thought i have to set from small to large size .....  so  now :1- resample in right way....  then in step-4  set large to small  124 32  8565 6561   is it correct?Message ID: ***@***.***>

**Reply by Xiaohua-Eric-Xu** [reply]
I am not sure I could read this correctly, but seems they have different size?

**Reply by coralraha1234** [reply]
Hi It was finally resolved thank you
